docs/user-guide/ze-profiles.md

```diff
@@ -42,6 +42,8 @@ Follow these steps:
 
 ### Associate Profile
 
+ > **Note**: The Associate Profile functionality is deprecated and will be removed in the version 2 of Zowe Explorer. Zowe Explorer V2 is slated for February 2022.
```

Comment:
> will be removed in the version 2 of Zowe Explorer.

I would remove "the" here: "will be removed in version 2 of Zowe Explorer."

Other than that, this looks good to me. Thanks @IgorCATech for this doc update!